### PR TITLE
periodic/openstack: add job for csi 4.8 every 72h

### PR DESCRIPTION
**Pull request by EmilienM**
Add a periodic job to test Cinder CSI every 3 days on OCP 4.8. This will
help to find potential regressions in the Cinder CSI driver operator.

[OSASINFRA-2605](https://issues.redhat.com/browse/OSASINFRA-2605)

### CODE REVIEW

**Review by vrutkovs — CHANGES_REQUESTED** [review]
>Container test is not ready with reason ImagePullBackOff and message Back-off pulling image "image-registry.openshift-image-registry.svc:5000/ci-op-jtj72y48/stable:openstack-installer"

Needs `openstack-installer` image defined for nightly 4.8

**Comment by jsafrane** [comment]
from periodic-ci-openshift-release-master-nightly-4.8-e2e-openstack-csi-cinder:

```
INFO[2021-08-02T13:37:00Z] pod didn't start running within 30m0s: 
* Container test is not ready with reason ImagePullBackOff and message Back-off pulling image "image-registry.openshift-image-registry.svc:5000/ci-op-rlf92j6g/stable:openstack-cinder-csi-driver-operator-test"
```

I think you need to add openstack-cinder-csi-driver-operator-test to the base images too.

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml [hunk 1]
```diff
@@ -51,6 +51,18 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: libvirt-installer
+  openstack-cinder-csi-driver-operator:
```
Comment (EmilienM) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:54]: @jsafrane do I need this one you think?

Comment (jsafrane) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:54]: No, it's not used by any CI step. The operator image is used by installer and it's baked into OCP release already. I think.

**Comment by jsafrane** [comment]
e2e-openstack-csi-cinder now installed a cluster successfully and failed snapshot tests, which seems to be genuine error either in the test or in the driver. The new job did what it was supposed to do.

The installation seems to be a bit flaky, but it's not problem of this PR.

/lgtm

**Comment by jsafrane** [comment]
/lgtm cancel
forgot about extra openstack-cinder-csi-driver-operator image, please remove it...

**Pull request closed without merging**